### PR TITLE
Update seed.ts in template

### PR DESCRIPTION
**Pull request by orta**
The default seed file types were quite right accurate, without `["data"]` it refers to the object with `{ select: {..}, data: {..} }` etc. This narrows that down.

Screen Shot 2021-12-08 at 12 13 16 PM

### CODE REVIEW

**Review by jtoar — APPROVED** [review]
LGTM! If I'm following this correctly, this is better because the previous type was really only one of the types `data` could actually be:

```ts
  /**
   * UserExample create
   */
  export type UserExampleCreateArgs = {
    /**
     * Select specific fields to fetch from the UserExample
     * 
    **/
    select?: UserExampleSelect | null
    /**
     * The data needed to create a UserExample.
     * 
    **/
    data: XOR<UserExampleCreateInput, UserExampleUncheckedCreateInput>
  }
```

File: packages/create-redwood-app/template/scripts/seed.ts [hunk 1]
```diff
@@ -20,7 +19,7 @@ export default async () => {
       // { name: 'bob', email: 'bob@example.com' },
     ]
     console.log(
-      "\nUsing the default './scripts/seed.js' template\nEdit the file to add seed data\n"
+      "\nUsing the default './scripts/seed.ts' template\nEdit the file to add seed data\n"
```
Comment (thedavidprice) [inline packages/create-redwood-app/template/scripts/seed.ts:22]: js|ts will be respective to the project language target. Should we do something like this here?
```
'./scripts/seed.{js,ts}'
```

Isn't that the "official" syntax for the extension option?

Comment (jtoar) [reply packages/create-redwood-app/template/scripts/seed.ts:22]: Yeah that would be more accurate. And yeah that's the correct syntax for path expansion. I can PR this real quick.

Comment (jtoar) [reply packages/create-redwood-app/template/scripts/seed.ts:22]: Oh you did it already; nice

**Comment by dac09** [comment]
I'm not so sure about removing the $. 

I think the import alias is configured to use the $api and $web. Maybe the tsconfig needs to be looked at instead.

**Comment by orta** [comment]
What is it used for? I use it without the `$` in this repo: https://github.com/orta/redwood-object-identification/blob/main/scripts/seed.ts (and it was the only `$` prefixed identifier in the codebase, I figured it was a leftover from a version of redwood where you prefixed import with `$`

**Comment by dac09** [comment]
Yeah it was meant to be a special use case for the scripts folder. I think it must be a side affect of the babel changes that it's working without the $api.

**Pull request merged**